README.md

```diff
@@ -45,7 +54,71 @@ const document = {
   language: 'fr',
 };
 
-botmeter.indexDocument(document, function(error, body) {
+genericLogger.indexDocument(document, function(error, body) {
+  // Body will equal 'ok' if logging is successful
   ...
 });
 ```
+
+### With Messenger
+
+```node.js
```

Comment:
Does this work ? I think you need javascript here to have colors and be in the right right pane column

Reply:
Oh this is for slate, probably not important here

Reply:
No you're right, I meant to write `javascript` 😃